### PR TITLE
add support for Velbus platform

### PR DESCRIPTION
**Pull request by thomasdelaet**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#8076

### CODE REVIEW

File: source/_components/velbus.markdown [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ha_iot_class: "Local Push"
+ha_release: 0.47
+---
+
+The `velbus` component supports the Velbus USB and Serial gateways.
+
+The gateway needs to be manually configured by adding the following lines to your `configuration.yaml` file:
+
+```yaml
+[velbus]
```
Comment (Landrash) [inline source/_components/velbus.markdown]: Your example greatly breaks from how other components are configured. Have a look at a few others.

File: source/_components/fan.velbus.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+To use your Velbus fans in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+
+fan:
+  - platform: velbus
+    devices:
+       - name: Light 1
```
Comment (Landrash) [inline source/_components/fan.velbus.markdown]: Page is for a fan and example for a light?

**Comment by thomasdelaet** [comment]
Done. Thanks.

**Comment by thomasdelaet** [comment]
@Landrash parent PR is merged now so would be great if you could merge this one.

**Comment by Landrash** [comment]
Thank you :cookie:

**Pull request merged**